single_file_stac_api/scripts/cli.py

```diff
@@ -0,0 +1,21 @@
+"""single_file_stac_api.cli"""
+import click
+
+from single_file_stac_api.backend import SingleFileClient
+from single_file_stac_api.server import Application
+
+
+@click.command()
+@click.argument("filepath", type=str)
+@click.option(
+    "--host",
+    type=str,
+    default="localhost",
+    help="Webserver host url (default: localhost)",
+)
+@click.option("--port", type=int, default=8005, help="Webserver port (default: 8005)")
+def api(filepath, host, port):
+    """start the api."""
+    client = SingleFileClient(filepath=filepath, host=f"http://{host}:{port}")
+    app = Application(client, host=host, port=port)
```

Comment:
I'm still not happy about the fact that the client (database) and the application needs to have the host information set. 

@geospatial-jeff do you think it's possible to dynamically edit the `base_url` in the Item when returning the object instead of doing it on import?